.github/workflows/test.yml

```diff
@@ -1,5 +1,8 @@
 name: Test
```

Comment:
Do we want to look to remove this? It seems like CircleCI is already handling the build step and this `test.yml` seems to be doing the same

Reply:
Checking CircleCI, seems like its not doing anything, we can look to remove the CircleCi config then as we aren't doing anything with it at a later time.